misc/systemd/cf-hub.service.in

```diff
@@ -6,8 +6,8 @@ ConditionPathExists=@workdir@/inputs/promises.cf
 After=syslog.target
 After=network.target
 
+Wants=cf-postgres.service
 After=cf-postgres.service
```

Comment:
I think we should change `Requires` to `Wants` in cf-apache as well since HA manages postgres outside of systemd and so no service should require cf-postgres.service.